schema.sql

```diff
@@ -13,3 +13,40 @@ CREATE TABLE animals (
 -- Add the "species" column to the "animals" table
 ALTER TABLE animals
 ADD species VARCHAR(255);
+
+-- Create the "owners" table
+CREATE TABLE owners (
+   id SERIAL PRIMARY KEY,
+   full_name VARCHAR(255) NOT NULL,
+   age INT NOT NULL
+);
+
+-- Create the "species" table
+CREATE TABLE species (
+   id SERIAL PRIMARY KEY,
+   name VARCHAR(255) NOT NULL
+);
+
+-- Add a new column 'species_id' of type INT
+ALTER TABLE animals
+ADD species_id INT;
+
+-- Add a new column 'owner_id' of type INT
+ALTER TABLE animals
+ADD owner_id INT;
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: :small_airplane: 

- [ ] I noticed your SQL statement for adding the `species_id` column to the `animals table`. It's great that you're working on this! However, to establish a foreign key relationship, it's important to specify the reference to the `species` table using the `REFERENCES` clause. This ensures data integrity and establishes the relationship between the two tables. :+1: :small_airplane: :small_airplane: 

Your query should look like this:
```sql
ALTER TABLE animals
ADD species_id INT REFERENCES species(id);
```
I'll suggest you update this for the `species_id` and `owner_id` :+1: